src/Shared/UnitTests/Vacancies.Client/Application/Services/EmployerServiceTest.cs

```diff
@@ -0,0 +1,159 @@
+using System.Threading.Tasks;
+using Esfa.Recruit.Vacancies.Client.Application.Services;
+using Esfa.Recruit.Vacancies.Client.Domain.Entities;
+using Esfa.Recruit.Vacancies.Client.Domain.Repositories;
+using FluentAssertions;
+using Moq;
+using Xunit;
+
+namespace Esfa.Recruit.UnitTests.Vacancies.Client.Application.Services
+{
+    public class EmployerServiceTest
+    {
+        private Mock<IEmployerProfileRepository> _mockEmployerProfileRepository = new Mock<IEmployerProfileRepository>();
+
+        [Theory]
+        [InlineData(VacancyStatus.Live)]
+        [InlineData(VacancyStatus.Submitted)]
+        [InlineData(VacancyStatus.Approved)]
+        [InlineData(VacancyStatus.Closed)]
+        public async Task GetEmployerNameAsync_ShouldReturnVacancyEmployerName(VacancyStatus status)
```

Comment:
Can we break this into two classes, one for each method

Reply:
👍